scatter-gather/scatter-gather.bal

```diff
@@ -0,0 +1,50 @@
+import ballerina/http;
+
+service /api/v1 on new http:Listener(8080) {
+    
+    resource function get vehicles(string dropOffDateTime, string dropOffLocation, string pickupDateTime, string pickupLocation) returns json|error {
+        return check requestRentalCar(dropOffDateTime, dropOffLocation, pickupDateTime, pickupLocation);
+    }
+}
+
+function transformResponse(map<anydata|error> responses) returns json|error {
```

Comment:
Let's break this to two functions `transformAvis` and `transformHotwire`. Then we can call it inside the each worker.